core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,16 @@
+const e2eBlogPostAdmin = require("./utility-functions/blogAdminUtils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+const blogDashboard = testConstants.URLs.BlogDashboard;
+
+async function deleteDraftAsBlogAdmin() {
+  const blogPostAdmin = await new e2eBlogPostAdmin();
+  await blogPostAdmin.getInitialized();
+
+  await blogPostAdmin.signInWithEmail("testadmin@example.com");
+  await blogPostAdmin.goto(blogDashboard);
+  await blogPostAdmin.deleteDraftBlog();
+  await blogPostAdmin.closeBrowser();
+}
```

Comment:
This test doesn't seem to have any expectations -- we should add some, e.g. that the blog post exists before deletion and that it doesn't after deletion.

Reply:
we can test creating a new draft blog post and then deleting a draft blog post in a single file.. both of them having the same title